turbopack/crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -2933,7 +2946,7 @@ impl ValueToString for ModulePart {
     async fn to_string(&self) -> Result<Vc<RcStr>> {
         Ok(Vc::cell(match self {
             ModulePart::Evaluation => "module evaluation".into(),
-            ModulePart::Export(export) => format!("export {}", export.await?).into(),
+            ModulePart::Export(export, _) => format!("export {}", export.await?).into(),
```

Comment:
Every argument need to be part of the resulting string. This generates the module id at runtime and need to be unique.